scripts.v3/utils.js

```diff
@@ -418,13 +407,48 @@ class ImporterExporter {
             }
 
             await this.httpClient.sendRequest("PUT", url, body);
-        }
-        catch (error) {
+        } catch (error) {
             throw new Error(`Unable to schedule website publishing. ${error.message}`);
         }
     }
+
+    /**
+     * Updates the url links of specified API Management service into updated url in destination.
+     */
+    async updateContentUrl(existingUrls, replaceableUrls) {
+        try {
+            if (existingUrls.Count != replaceableUrls.Count) {
```

Comment:
`existingUrls.Count` - property `Count` doesn't exist in standard JS arrays. What kind of type is expected here? Can we use `length` instead?

Reply:
Fixed it and tested it.